src/rgw/rgw_perf_counters.h

```diff
@@ -4,11 +4,20 @@
 #pragma once
 
 #include "include/common_fwd.h"
+#include "common/perf_counters_cache.h"
 
 extern PerfCounters *perfcounter;
+extern PerfCountersCache *perf_counters_cache;
 
 extern int rgw_perf_start(CephContext *cct);
 extern void rgw_perf_stop(CephContext *cct);
+extern void add_rgw_put_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_get_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_del_obj_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_del_bucket_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_copy_obj_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_list_obj_counters(PerfCountersBuilder* lpcb);
+extern void add_rgw_list_buckets_counters(PerfCountersBuilder* lpcb);
```

Comment:
as i mentioned in the weekly refactoring call, i think we're better off combining these into a single instance. if one bucket is "hot", i think we'd want to keep all of its counters instead of letting some of them get evicted

p.s. it looks like these functions are only used inside of rgw_perf_counters.cc so we don't need to expose them in the header